src/node/optimizer/scan.ts

```diff
@@ -228,7 +228,7 @@ async function transformDynamicRequire(
 			continue;
 		}
 
-		const url = source.slice(start, end);
+		const url = source.slice(start, end).replace(/,$/, '');
```

Comment:
@janvennemann I feel it may remove all trailing comma for each line, even for array or dicts?

Reply:
This will only be used for `require` calls which are parsed in the code above from a files's source code with `parseRequire`. This will not be applied to any other code.